src/Concordium/Client/Output.hs

```diff
@@ -699,6 +735,19 @@ showEvent verbose = \case
     showDelegationTarget Types.DelegatePassive = "Passive delegation"
     showDelegationTarget (Types.DelegateToBaker bid) = "Baker ID " ++ show bid
 
+    showLoggedEvents :: Maybe SchemaType -> [Wasm.ContractEvent] -> String
+    showLoggedEvents st evs = "\n" <> case evs of
+      [] -> "No contract events were logged in this transaction."
```

Comment:
How should they differ.

To my knowledge the output of `show transaction` already states the event type. `showLoggedEvents` is currently only called from `showEvent` in case of `ContractInitialized`, `Interruped` or `Update`.

Reply:
For the `ContractInitialized` this message is fine.

For when this is applied to the `Interrupt` then it is not correct. The logs in that event are only for the section of the code before the interrupt. It is a bit hard to state this very precisely since the exact condition is complex, but something like `"No contract events were emitted." is probably fine.

I would not mention "transaction" for Interrupted and Updated events.